src/main/java/hu/oe/nik/szfmv/automatedcar/model/XmlConverter.java

```diff
@@ -0,0 +1,185 @@
+package hu.oe.nik.szfmv.automatedcar.model;
+
+import hu.oe.nik.szfmv.automatedcar.model.objects.CrossWalk;
+import hu.oe.nik.szfmv.automatedcar.model.objects.ParkingPlace;
+import hu.oe.nik.szfmv.automatedcar.model.objects.Tree;
+import hu.oe.nik.szfmv.automatedcar.model.objects.Road;
+import hu.oe.nik.szfmv.automatedcar.model.objects.RoadSign;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.w3c.dom.Document;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.w3c.dom.Element;
+import org.xml.sax.SAXException;
+
+import javax.xml.crypto.dsig.XMLSignatureException;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class XmlConverter {
```

Comment:
XmlParser would be a more suitable name.

Reply:
Solved